CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+* Remove `garcon` from API.
+
```

Comment:
Since this is a breaking change to the API, can you say something here about what changes will be required?  In this case it should be simple since it's just `Waiter` parameters being removed/no longer necessary for callers.